grakn-engine/src/main/java/ai/grakn/engine/loader/LoaderTask.java

```diff
@@ -143,7 +143,7 @@ private void throwException(String message, Collection<InsertQuery> inserts){
      * @param e exception to log
      * @param i amount of time to sleep
      */
-    private void handleError(Throwable e, int i) {
+    private void handleError(Throwable e, long i) {
```

Comment:
Okay but realistically if you fail more then 2147483647 times then you probably not going to succeed on the 2147483648th time. Surely at some point we say enough enough and just log the permanent failure?